packages/theme-chalk/src/radio.scss

```diff
@@ -17,6 +17,7 @@
   cursor: pointer;
   display: inline-flex;
   align-items: center;
+  vertical-align: middle;
```

Comment:
Do you add `vertical-align` to decrease the height of radio group?

Reply:
The radio group is `inline-block`, and the radio is `inline-flex`, should we use `inline-flex` for radio group?

If we do not need, just merge it.

Reply:
Yes. I think both are fine.